suite-native/module-dev-utils/src/screens/DevUtilsScreen.tsx

```diff
@@ -84,12 +104,17 @@ export const DevUtilsScreen = ({
                             />
                         </Card>
 
+                        <Button onPress={() => setAreLogsVisible(!areLogsVisible)}>
```

Comment:
You can make this checkbox like two items above for renredring, it makes more sense if it on/off

Reply:
https://github.com/trezor/trezor-suite/pull/7579/commits/4b4d8ecea9a82dd812770c861be80f974433af87

<img width="403" alt="image" src="https://user-images.githubusercontent.com/36101761/217516695-e3abe2ca-051a-461f-a47d-e100a688bb6a.png">